include/eve/detail/function/simd/common/broadcast.hpp

```diff
@@ -14,13 +14,31 @@ namespace eve::detail
   template<simd_value Wide>
   EVE_FORCEINLINE auto broadcast_(EVE_SUPPORTS(cpu_), Wide v, auto Index) noexcept
   {
-    return Wide{ v.get(Index) };
+    if constexpr( is_bundle_v<typename Wide::abi_type> )
+    {
+      return Wide(kumi::map( [&]<typename T>(T m) { return broadcast(m,Index); }, v.storage()));
```

Comment:
Hmm, can we like make a standard helper to do this?

Reply:
Will propagate required kumi functions as friend of wide